lib/facter/swap_file_size.rb

```diff
@@ -0,0 +1,19 @@
+Facter.add('swap_file_size') do
```

Comment:
Don't worry, I have a hacky way to get non-strinified working with some csv magic 👍